packages/compass-web/scripts/electron-proxy.js

```diff
@@ -213,67 +242,116 @@ class AtlasCloudAuthenticator {
     })());
   }
 
-  async logout() {
-    await session.defaultSession.clearStorageData({ storages: ['cookies'] });
+  async cleanupAndLogout() {
+    // When logging out, delete the test user too. If we don't do this now, we
+    // will loose a chance to do it later due to missing auth
```

Comment:
grammar nit: lose, not loose. loose is an adjective that means not firmly attached 😄

Reply:
Thanks! Fixed in both places where I made this mistake 😅